pkg/cloud/cloud.go

```diff
@@ -224,7 +224,7 @@ type EC2 interface {
 	DescribeAvailabilityZonesWithContext(ctx aws.Context, input *ec2.DescribeAvailabilityZonesInput, opts ...request.Option) (*ec2.DescribeAvailabilityZonesOutput, error)
 }
 
-type Cloud interface {
+type AWSCloud interface {
```

Comment:
fair enough. note that i do eventually want to reconcile our cloud implementation with the generic aws cloudprovider implementation https://github.com/kubernetes/cloud-provider-aws/blob/master/pkg/providers/v2/cloud.go#L63  so "Cloud" is not overly ambiguous, the current naming is probably fine.  https://github.com/kubernetes-sigs/aws-ebs-csi-driver/issues/806#issuecomment-823535745

Reply:
my IDE was complaining about Cloud being declared here and in a 3P dependency, but that went away so I don't rename it in PR #855